XLS-0070d-credentials/README.md

```diff
@@ -0,0 +1,573 @@
+
+This proposal only supports blocking the interactions that Deposit Authorization supports, such as direct payments. However, future proposals could add support for credential-gating in other parts of the XRPL, such as AMMs or lending pools.
+
+We propose:
+* Creating a `Credential` ledger object
+* Modifying the `DepositPreauth` ledger object
+* Modifying the `DepositPreauth` transaction
+* Creating a `Credential` ledger object
+* Creating a `CredentialCreate` transaction type
+* Creating a `CredentialAccept` transaction type
+* Creating a `CredentialDelete` transaction type
+
+This feature will require an amendment, tentatively titled `featureCredentialAuth`.
+
+### 1.1. Background: DIDs and Verifiable Credentials (VCs)
+
+A [verifiable credential](https://en.wikipedia.org/wiki/Verifiable_credentials) (VC), as defined by the [W3C specification](https://www.w3.org/TR/vc-data-model-2.0/), is a secure and tamper-evident way to represent information about a subject, such as an individual, organization, or even an IoT device. These credentials are issued by a trusted entity and can be verified by third parties without directly involving the issuer at all.
+
+For a clearer understanding of the relationship between [DIDs (Decentralized Identifiers)](https://en.wikipedia.org/wiki/Decentralized_identifier) and VCs, a real-world example is considered. A DID serves as a unique identifier for an entity, similar to a fingerprint or a photo of a person (their physical characteristics). In contrast, a VC acts as a verifiable piece of information associated with that DID, much like a driver's license or passport. When a third party needs to verify an identity claim, they can examine the VC presented by the entity and ensure it aligns with the corresponding DID (such as comparing the photo on a driver's license to the physical characteristics of the person to whom it supposedly belongs).
+
+### 1.2. Terminology
+
+* **Credential**: A representation of one or more assertions made by an issuer. For example, a passport is a representation of the assertion that a person is a citizen of a given country.
+* **Issuer**: The account that creates (issues) the credential. For example, a passport is issued by a country's government.
+* **Subject**: The account that the credential is issued to. For example, a passport is issued to a specific person.
+* **Whitelist**: A list of accounts that are allowed to do a given action, or (if used as a verb) the action of adding an account to said list.
+* **Preauthorization**: Essentially the same thing as whitelisting.
+
+*Note: These definitions do not directly match the definitions listed in the [W3C Verifiable Credentials spec](https://www.w3.org/TR/vc-data-model-2.0/#terminology). These terms are used in a slightly different way in the context of this spec.* 
```

Comment:
Howso? Aside from the presence of some terms that aren't in that list, I don't see an obvious conflict between how the terms are used here and there.

Reply:
For example, credentials can refer to many subjects, and the credentials described here aren't exactly verifiable credentials as described in the spec. Also, a subject in this case must be an on-ledger account - you can't make a credential for some off-chain entity, for example.